### PR TITLE
fixed Entity.Components returns one same component

### PR DESCRIPTION
**Pull request by floatW0lf**
Entity.Components returns one same component.

### CODE REVIEW

File: src/EcsRx.Tests/Framework/EntityTests.cs [hunk 1]
```diff
@@ -212,5 +212,25 @@ public void should_add_components_in_parameter_order()
                 componentDatabase.Set(2, 1, fakeComponents[2]);
             });
         }
+
+        [Fact]
+        public void should_entity_components_returns_not_one_same_component()
```
Comment (grofit) [inline src/EcsRx.Tests/Framework/EntityTests.cs]: Can we rename this to something more generic as the test is really just confirming all components are returned correctly
```suggestion
        public void should_return_all_components_allocated_to_the_entity()
```

**Comment by grofit** [comment]
Another great catch, the last refactor required most tests re-writing so I skimped on some of them, good thing you are here to pick them up, I have one recommendation on the naming, if you are happy enough to change it I am happy enough to approve it.

File: src/EcsRx.Tests/Framework/EntityTests.cs (hunk 1 above)
Comment (floatW0lf) [reply src/EcsRx.Tests/Framework/EntityTests.cs]: yes we can

**Comment by grofit** [comment]
Thanks again @floatW0lf

**Pull request merged**